client/main.lua

```diff
@@ -157,15 +157,23 @@ local function DeliverAnim()
 end
 
 function TakeAnim()
-    exports.qbx_core:Progressbar("bag_pickup", Lang:t("info.picking_bag"), math.random(3000, 5000), false, true, {
-        disableMovement = true,
-        disableCarMovement = true,
-        disableMouse = false,
-        disableCombat = true,
-    }, {
-        animDict = "anim@amb@clubhouse@tutorial@bkr_tut_ig3@",
-        anim = "machinic_loop_mechandplayer",
-        flags = 16,
+    lib.progressbar({
+        name = 'bag_pickup',
+        label = Lang.t("info.picking_bag"),
+        duration = math.random(3000, 5000),
+        useWhileDead = false,
+        canCancel = true,
+        disable = {
+            move = true,
+            car = true,
+            mouse = false,
+            combat = true,
+        },
+        animation = {
+            dict = 'anim@amb@clubhouse@tutorial@bkr_tut_ig3@',
+            anim = 'machinic_loop_mechandplayer',
+            flags = 16,
+        },
```

Comment:
There's no need to do this, because the Progressbar method in qbx_core is a wrapper for ox-lib progress bar.
https://github.com/Qbox-project/qbx_core/blob/d60f0be936a5fbbc952d97ae00dd43acbe8bf0c7/bridge/qb/client/functions.lua#L40